static/app/views/issueList/issueListTable.tsx

```diff
@@ -76,6 +79,22 @@ function IssueListTable({
     return <NewViewEmptyState />;
   }
 
+  function getMinHeight() {
```

Comment:
Displaying a min height works, but is quite fragile (if we change the size of anything, we would need to remember to change the hardcoded values here which will probably be forgotten about). Can we instead continue displaying the skeleton loaders in the case where `!groupIds.length && statsLoading`?

Another way to fix this is to ensure that `groupIds` and `issuesLoading` is synced better (it sounds like `groupIds` takes a render cycle to have the correct value?) That would be the best solution, but I'm not sure what the cause of the mismatch is so it could be difficult to fix